_authentication.md

```diff
@@ -138,12 +148,12 @@ token     | yes      | The PAT you wish to revoke.
 
 ```bash
 curl https://api.uphold.com/v0/me \
-  -u 41ee8b1fa14042e031fe304bb4793b54e6576d19b306dc205136172b80d59d20:X-OAuth-Basic
+  -H "Authorization: Bearer <token>"
 ```
 
-A PAT may be used for authenticating a request via the HTTP Basic Authentication scheme.
+A PAT may be used for authenticating a request via the oAuth scheme.
```

Comment:
OAuth, as used elsewhere in this document.

Reply:
Fixed.